hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyRenameRequest.java

```diff
@@ -327,11 +331,13 @@ private OMRequest doPreExecute(OMRequest originalOmRequest) throws Exception {
    * @return OMRequest
    */
   private OMRequest createRenameKeyRequest(String toKeyName) {
-    KeyArgs keyArgs = KeyArgs.newBuilder().setKeyName(keyName)
+    Map<String, String> renameKeyMap = new HashMap<>();
```

Comment:
General comment for both `delete and rename `batch apis.

Please add some test cases with `all, partial, no success` behaviors. Thanks!